src/app/CometApplication.cpp

```diff
@@ -386,13 +417,19 @@ vector<string> CometApplication::getOptions() const {
     "num_results",
     "skip_researching",
     "max_fragment_charge",
+    "max_index_runtime",
     "max_precursor_charge",
     "nucleotide_reading_frame",
     "clip_nterm_methionine",
     "spectrum_batch_size",
     "decoy_prefix",
     "output_suffix",
+    "peff_verbose_output",
+    "peptide_length_range",
+    "precursor_NL_ions",
+    "equal_I_and_L",
     "mass_offsets",
+    "max_duplicate_proteins",
     // Spectral processing
     "minimum_peaks",
     "minimum_intensity",
```

Comment:
Wow, there were a bunch of new parameters!

Reply:
Yes, there certainly were! Most of them required trivial modifications. The only one that required any real changes in our code was search_enzyme2_number, which added the ability to specify a 2nd digestion enzyme.